PlannTer/Modules/PlantModule/PlantAddView.swift

```diff
@@ -4,6 +4,7 @@ import SwiftData
 struct PlantAddView: View {
     @Environment(\.modelContext) private var context
     @Bindable var room: RoomModel
+    @Query var roomList: [RoomModel]
```

Comment:
it may be considered nitpicking bud `room`, `roomList`, `rooms` and `selectedRoom` may be a bit confusing

Reply:
imma leave it as is for now, they are all needed for functionality